pkg/build/options.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package build
+
+import (
+	"fmt"
+)
+
+type Options struct {
+	Strategy           string `flag:"~strategy" desc:"Determines the build strategy to use either make or bazel."`
```

Comment:
nit: seems like this could be its own type, but won't block

Reply:
They are separate types: 
https://github.com/kubernetes-sigs/kubetest2/blob/03dbeda27f3fc7a7629d6a01e12e8174eceee6a5/pkg/build/make.go#L25-L27

https://github.com/kubernetes-sigs/kubetest2/blob/03dbeda27f3fc7a7629d6a01e12e8174eceee6a5/pkg/build/bazel.go#L27-L31

the string here is to expose the available strategies as a flag.

Reply:
most flag libraries allow you to use the real type instead of a string /shrug

Reply:
the upside to doing so is that you can then pass around this struct as the real parsed values

Reply:
we are using the real type which is `BuildOptions` and has the actual options like RepoRoot , but at some point you need to specify whether it's bazel or make :)

exposing the individual implementation directly would mean weirdly exposing individual flags like --make-repo-root and --bazel-repo-root which looks worse /shrug

Reply:
I guess I could have been clearer and suggested a string-as-enum approach (`type BuildStrategy string` etc.) but as-is is fine, like I said, won't block.